checkin/models.py

```diff
@@ -11,12 +20,30 @@ def __str__(self):
         return self.title
 
     def locale_title(self, language_code):
+        """Returns skill name in correct language.
```

Comment:
Personally I am not a big fan of using returns here, as that generally means a repeat of what is said at `:return:`. Should rather be something like "Finds the title of the skill in the given language". Should also be combined with the line below.